encoder/src/main/java/com/pedro/encoder/input/gl/render/ScreenRender.java

```diff
@@ -131,6 +131,8 @@ public boolean isAAEnabled() {
     return AAEnabled;
   }
 
+  public boolean isPortrait() { return isPortrait; }
```

Comment:
@SouOka 
なるべくフォーマットはあわせておきたいかん
```suggestion
  public boolean isPortrait() {
      return isPortrait;
  }
```

Reply:
@mn-D128 
12bcc07 で対応